src/NotificationDbService.js

```diff
@@ -1,10 +1,16 @@
 const { Notification } = require('./database/mongooseSchema')
 
 module.exports = {
+	saveNotification: async notification => {
+		const res = await Notification.create(notification)
+		return res
+	},
 	saveNotifications: async notifications => {
 		try {
-			const savedNotificationLogs = await Notification.insertMany(notifications)
-			return savedNotificationLogs
+			for (const notification of notifications) {
+				const res = await Notification.create(notification)
```

Comment:
@shrijan-siris we should use `Promise.all` to Not wait for each save in sequence